tests/libtest/test_ibmcloud.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf8 -*-
+
+
+import logging
+
+from ocs_ci.ocs.platform_nodes import IBMCloud
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import libtest
+from ocs_ci.ocs import constants, node
+from ocs_ci.ocs.node import get_all_nodes
+from ocs_ci.framework.pytest_customization.marks import ibmcloud_platform_required
+
+from ocs_ci.utility.utils import run_cmd
+
+logger = logging.getLogger(__name__)
+
+
+@libtest
+@ibmcloud_platform_required
+def test_rebootnodes():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+    worker_nodes = node.get_nodes(node_type="worker", num_of_nodes=1)
+    ibmcloud.restart_nodes(worker_nodes)
+
+@libtest
+@ibmcloud_platform_required
+def test_attachvolume():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+
+    volume = ibmcloud.getVolumeId()
+
+    worker_nodes = node.get_nodes(node_type="worker", num_of_nodes=1)
+    ibmcloud.attach_volume(volume, worker_nodes)
+
+@libtest
+@ibmcloud_platform_required
+def test_detachvolume():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+
+    volume = ibmcloud.getVolumeId()
+
+    worker_nodes = node.get_nodes(node_type="worker", num_of_nodes=1)
+    ibmcloud.detach_volume(volume, worker_nodes)
+
+@libtest
+@ibmcloud_platform_required
+def test_get_node_by_attached_volume():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+
+    volume = ibmcloud.getVolumeId()
+    ibmcloud.get_node_by_attached_volume(volume)
+
+@libtest
+@ibmcloud_platform_required
+def test_get_data_volumes():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+
+    ibmcloud.get_data_volumes()
+
+@libtest
+@ibmcloud_platform_required
+def test_wait_for_volume_attach():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+
+    volume = ibmcloud.getVolumeId()
+
+    ibmcloud.wait_for_volume_attach(volume)
```

Comment:
The same fixture for creating this volume can be used here.

The thing is that the tests should not be depending on each other and I should be able to run only one specific test and the things which are not part of test itself should be handled in setup/teardown(finalizer) of fixture.